src/material/datepicker/date-range-input-parts.ts

```diff
@@ -187,8 +185,6 @@ const _MatDateRangeInputBase:
     '(change)': '_onChange()',
     '(keydown)': '_onKeydown($event)',
     '[attr.id]': '_rangeInput.id',
```

Comment:
I think we need also remove that from here and add the id to the actual range input. In the current state it will announce (for ChromeVox) the label, and then mention edit text group w/ label again.

Reply:
I think that associating a label with a non-input element is invalid. Also the idea here was to allow for clicking on the `label` to move focus automatically to the start input.

Reply:
I think you're right that the label `for` would not work for a `role="group"`. Might be a general issue in our form-field as we always set the label `for` while we support custom controls that don't necessarily use a [`labelable` element](https://www.w3.org/TR/html52/sec-forms.html#labelable-element).

The focus for the start input should still work through the form-field `onContainerClick`, so that might not be an issue here. I think we should be good with the latest changes, but we might need to revisit this when we think more about how we want to handle the `<label for>` attribute in form fields.

Reply:
@crisbeto we do the same already for the select where `label` points to a custom control. That is a noop for screenreaders, but definitely not 100% valid markup. Also the select might not be the best candidate for accessibility comparisons. I'll leave that up to you. I don't feel too strongly about it, but would definitely prefer label not being announced twice.